### PR TITLE
Improve Marathon LB Documentation.

### PR DESCRIPTION
**Pull request by orsenthil**
Specify Accurately the kind of Marathon-LB options demonstrated in the
tutorial.

## What type of changes does your doc introduce?
- [x] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [ ] I have built locally and tested for broken links and formatting.
- [ ] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [x] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

`@emanic`, `@sascala`, or `@joel-hamill` - Please see if this is suitable.

## If you included a comment with your commit, it appears here:

### CODE REVIEW

**Comment by orsenthil** [comment]
With this change, I think, we can close this ticket (https://mesosphere.atlassian.net/browse/DOCS-1192)

File: 1.8/usage/service-discovery/marathon-lb/index.md [hunk 1]
```diff
@@ -12,20 +12,25 @@ Up to date documentation for Marathon-LB can be found on the GitHub page.
...
 
-*   Use Marathon-LB as your edge load balancer and service discovery mechanism. You could run Marathon-LB on public-facing nodes to route ingress traffic. You would use the IP addresses of your public-facing nodes in the A-records for your internal or external DNS records (depending on your use-case).
-*   Use Marathon-LB as an internal LB and service discovery mechanism, with a separate HA load balancer for routing public traffic in. For example, you may use an external F5 load balancer on-premise, or an Elastic Load Balancer on Amazon Web Services.
-*   Use Marathon-LB strictly as an internal load balancer and service discovery mechanism.
-*   You might also want to use a combination of internal and external load balancers, with different services exposed on different load balancers.
+1. Use Marathon-LB as your edge load balancer and service discovery mechanism. You could run Marathon-LB on public-facing nodes to route ingress traffic. You would use the IP addresses of your public-facing nodes in the A-records for your internal or external DNS records (depending on your use-case).
```
Comment (joel-hamill) [inline 1.8/usage/service-discovery/marathon-lb/index.md]: Generally a numbered list should imply a numerical order (e.g. steps). In this case I think we are just showing options?

File: 1.8/usage/service-discovery/marathon-lb/index.md [hunk 2]
```diff
@@ -12,20 +12,25 @@ Up to date documentation for Marathon-LB can be found on the GitHub page.
...
+2. Use Marathon-LB as an internal LB and service discovery mechanism, with a separate HA load balancer for routing public traffic in. For example, you may use an external F5 load balancer on-premise, or an Elastic Load Balancer on Amazon Web Services.
+3. Use Marathon-LB strictly as an internal load balancer and service discovery mechanism.
+4. You might also want to use a combination of internal and external load balancers, with different services exposed on different load balancers.
 
-Here, we discuss the fourth option above in order to highlight the features of Marathon-LB.
+Here, we discuss the option 1. and option 3. above in order to highlight the features of Marathon-LB.
```
Comment (joel-hamill) [inline 1.8/usage/service-discovery/marathon-lb/index.md]: `Here we discuss Marathon-LB as an edge load balancer and as an internal and external load balancer.`

File: 1.8/usage/service-discovery/marathon-lb/index.md [hunk 3]
```diff
@@ -12,20 +12,25 @@ Up to date documentation for Marathon-LB can be found on the GitHub page.
...
+
+## Marathon-lb as an edge load balancer
```
Comment (joel-hamill) [inline 1.8/usage/service-discovery/marathon-lb/index.md]: `Marathon-LB`

File: 1.8/usage/service-discovery/marathon-lb/index.md [hunk 4]
```diff
@@ -12,20 +12,25 @@ Up to date documentation for Marathon-LB can be found on the GitHub page.
...
 
 ![lb1](img/lb1.png)
 
+## Marathon-lb as an internal and external load balancer
```
Comment (joel-hamill) [inline 1.8/usage/service-discovery/marathon-lb/index.md]: `Marathon-LB`

File: 1.9/usage/service-discovery/marathon-lb/index.md [hunk 5]
```diff
@@ -12,20 +12,26 @@ Up to date documentation for Marathon-LB can be found on the GitHub page.
```
Comment (joel-hamill) [inline 1.9/usage/service-discovery/marathon-lb/index.md]: Please make similar changes as suggested in 1.8.

**Comment by orsenthil** [comment]
@joel-hamill , all comments are addressed. Thanks.

**Comment by joel-hamill** [comment]
@orsenthil where did this graphic come from? it seems familiar - wondering if it's reused? i'll file a request with the design team to beautify.

**Comment by orsenthil** [comment]
@joel-hamill its from the our blog post ( https://mesosphere.com/blog/2015/12/04/dcos-marathon-lb/), and yeah, design team can probably improve it and we can include more consistent one. 

I thought about it raising  request with design team and then creating PR, then I did not want to blocked on some effort when an existing improvement is possible with  graphic available from the blog.

**Comment by joel-hamill** [comment]
@orsenthil  i think we should hold off on posting this graphic until the design team can revise, otherwise the page is going to look very inconsistent visually: https://cl.ly/3d283k3J1f2v. i have created the ticket and will follow up with the team to accelerate (https://mesosphere.atlassian.net/browse/DESIGN-1858).

**Comment by orsenthil** [comment]
Sounds good to me. Making the change after getting an updated design will be very easy.

**Pull request merged**